### PR TITLE
[CI] Quick change: install ignite from github releases

### PR DESCRIPTION
**Pull request by okdas**
Quick PR to address an ongoing issue ignite experiences with CloudFlare.

### CODE REVIEW

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -14,7 +14,9 @@ jobs:
     steps:
       - name: install ignite
         run: |
-          curl https://get.ignite.com/cli! | bash
+          wget https://github.com/ignite/cli/releases/download/v0.27.1/ignite_0.27.1_linux_amd64.tar.gz
```
Comment (okdas) [inline .github/workflows/go.yml:17]: Also pins to the version we currently use locally.

Comment (Olshansk) [reply .github/workflows/go.yml:17]: Can you #PUC that this pins the version and we can use `curl https://get.ignite.com/cli! | bash` (when it works) for the latest & greatest?

**Review by Olshansk — APPROVED** [review]
@okdas #PUC (Please Update Comment) with details and alternatives, but other than that, 🚢

**Comment by okdas** [comment]
Closing this, as ignite.com now works. Will add a comment linking to this PR in case a similar issue arises.

**Pull request closed without merging**